Dockerfile.rhel

```diff
@@ -1,34 +1,27 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal
+# This is an exact copy of the Dockerfile except that it defaults
+# to using the Red Hat UBI base image.  In the future we can switch
+# to use the standard Dockerfile for all builds using BASE_IMAGE.
 
-ARG TARGETARCH
+ARG BASE_IMAGE=registry.access.redhat.com/ubi8/ubi-micro
 
-# ubi-minimal updates frequently and has very few packages installed,
-# so no need for a "security-only" update.
-RUN microdnf update && microdnf clean all
+FROM ${BASE_IMAGE}
 
-# Add licenses and help file
-RUN mkdir /license
-COPY LICENSE /licenses/LICENSE.txt
-COPY README.md /help.1
```

Comment:
Is it "OK" that you're no longer including the README? It looks like the exact filename `/help.1` isn't required (although it IS mentioned in Red Hat's documentation), but shouldn't it still be in there somewhere? Ditto for the non-RHCC image.